modules/app.xql

```diff
@@ -59,6 +61,12 @@ declare function app:scheduler-enabled($node as node(), $model as map(*)) {
         $node
 };
 
+declare 
+    %templates:warp
+function app:java-version ($version as xs:string?, $model as map(*)) as node()? {
```

Comment:
Ah, I see - yes, in order to be able to call a function from an HTML template, the function requires there to be two parameters - `$node` and `$model`. From the templating docs:

> The two required parameters here are `$node` and `$model`. `$node` contains the HTML node currently being processed: in this case, the div element. `$model` is an XQuery map with application data. 

I guess the templating modules weren't quite sensitive enough to alert you about having changed the name/signature of the first parameter. Sorry for not spotting this myself!

Your most recent change is the way to go on this.